modopt/tests/test_base.py

```diff
@@ -275,3 +277,53 @@ def test_check_npndarray(self):
             self.data3,
             dtype=np.integer,
         )
+
+
+class TestBackend(TestCase):
+    """Test the backend codes."""
+
+    def setUp(self):
+        """Set test parameter values."""
+        self.input = np.array([10, 10])
+
+    @skipIf(LIBRARIES['tensorflow'] is None, 'tensorflow library not installed')
+    def test_tf_backend(self):
+        """Test tensorflow backend."""
+        xp, backend = get_backend('tensorflow')
+        if backend != ':tensorflow' or xp != LIBRARIES['tensorflow']:
```

Comment:
This may be system specific, but on macOS the backend returned is just `tensorflow`.

```suggestion
        if 'tensorflow' not in backend or xp != LIBRARIES['tensorflow']:
```

Reply:
I got this wrong, Its a typo, I wonder why its passing in linux